modules/working-in-code-server.adoc

```diff
@@ -0,0 +1,67 @@
+:_module-type: PROCEDURE
+
+[id="working-in-code-server_{context}"]
+= Working in code-server
+
+[role='_abstract']
+{productname-long} includes the code-server workbench image. The code-server IDE provides the open-source core of link:https://github.com/Microsoft/vscode[VS Code] and allows you to run it in a web browser.
+
+For more information on code-server, see link:https://github.com/coder/code-server[code-server in GitHub].
+
+NOTE: Elyra-based pipelines are not available with the code-server workbench image. 
+
+
+ifndef::upstream[]
+[IMPORTANT]
+====
+ifdef::self-managed[]
+The code-server notebook image is currently available in {productname-long} {vernum} as a Technology Preview feature.
+endif::[]
+ifdef::cloud-service[]
+The code-server notebook image is currently available in {productname-long} as a Technology Preview feature.
+endif::[]
+Technology Preview features are not supported with Red{nbsp}Hat production service level agreements (SLAs) and might not be functionally complete.
+Red{nbsp}Hat does not recommend using them in production.
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+endif::[]
+
+
+== Installing extensions with code-server
+
+With the code-server workbench image, you can customize your code-server environment by adding extensions that add new languages, themes, debuggers, and connect to additional services. You can also enhance the efficiency of your data science work with extensions for syntax highlighting, auto-indentation, and bracket matching. 
```

Comment:
A bit confusing. Maybe something like: 
_With the code-server workbench image, you can customize your code-server environment by using extensions to add new languages, themes, and debuggers, and to connect to additional services._